contrib/native/client/src/test/DrillClientTest.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Could you please fix license here to fix Travis job? You may copy it from `contrib/native/client/example/querySubmitter.cpp` and paste with preserving all trailing spaces.

Reply:
@vvysotskyi - i did check both files [this and contrib/native/client/example/querySubmitter.cpp] and License in both files are exactly same. 
Is there any thing else due to which travis might have failed?

Reply:
Perhaps the problem with spaces before every line after the first one, please try to add them and will see whether it would help:
```suggestion
 * Licensed to the Apache Software Foundation (ASF) under one
```

Reply:
@vvysotskyi - the build passed after the change. This was a great fix - thanks for your help.
But may I know why this whitespace in License comments made the build fail?

Reply:
Travis build includes some additional checks, one of them is license check. We use two plugins for licenses: rat and license-maven-plugin. They both require a strict match of the file headers for every specific file extension.

Reply:
Okay, thanks for the information.